### ISSUE TITLE
Unable to Render graph using RAST Annotations

### ISSUE THREAD

**Issue by jdakota1305** [labgem/PPanGGOLiN#44]
Hi All,

I am currently having an issue when I try using my own annotations that were generated using RAST. Is RAST a supported annotator? I am noticing some differences in the .gbk file formatting for my RAST annotations when compared to NCBI .gbk files (which I have gotten to work successfully). 

I can provide the files if it would be helpful. 

Thank you!

**Comment by axbazin** [comment]
Hi,

It would be extremely helpful if you could provide the files (or to tell me where I can find them) so I can check this.
I've not tested with RAST before so there might be specificities in their files that break our parser.

Thank you :)

**Comment by jdakota1305** [comment]
Hello, here is an example of a RAST annotation we have tried using. 
[GCF_000158275.2_ASM15827v2_genomic.fna.zip](https://github.com/labgem/PPanGGOLiN/files/5513651/GCF_000158275.2_ASM15827v2_genomic.fna.zip)

**Comment by axbazin** [comment]
This is a fna file, I would actually need the .gbk files that you generated to test it, if that is possible

**Comment by jdakota1305** [comment]
Sorry for the delay-- here is a .gbk file
[FN_ATCC10953.zip](https://github.com/labgem/PPanGGOLiN/files/5570534/FN_ATCC10953.zip)

**Comment by axbazin** [comment]
No problem, thank you !
I will have a look into this

**Comment by axbazin** [comment]
The annotations in the file you provided seem to be perfectly loaded when I run ppanggolin on it. The only problem I noticed is the way unique gene identifiers are saved in the .gbk files from RAST, however it did not raise any error. I fixed the problem with the gene IDs.

I would need more details on your issue to be able to help, like the logs of the ppanggolin analysis that failed with the RAST annotations for example, or at least the error that was raised, if there were any.
